rma_sale/security/ir.model.access.csv

```diff
@@ -1,3 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_sale_order_rma_wizard_user_all,sale.order.rma.wizard.user.all,model_sale_order_rma_wizard,rma.rma_group_user_all,1,1,1,1
+access_sale_order_rma_wizard_portal_all,sale.order.rma.wizard.portal.all,model_sale_order_rma_wizard,base.group_portal,1,1,1,1
```

Comment:
Wouldn't a `sudo` fit this issue better? We're giving portal users whole rights to that model :thinking:

Reply:
This is just about a wizard, do you think this could be a security issue?

Reply:
Please change to sudo as commented by chat